README.md

```diff
@@ -55,7 +62,15 @@ A customizable Neovim plugin to run code inside the editor
 
 ## Configuration
 
-  This plugin doesn't have a `setup()` method, but you can set your filetype handlers using the `set_handler()` method.
+  ### S`setup(options)`
+  Runner comes with the following defaults:
+  ```lua
+  {
```

Comment:
I would prefer you run the setup function in this block to make it more explicit and to have a copy-pastable example.